pkg/ccloudv2/sso.go

```diff
@@ -2,11 +2,9 @@ package ccloudv2
 
 import (
 	"context"
-	"net/http"
-
 	ssov2 "github.com/confluentinc/ccloud-sdk-go-v2/sso/v2"
-
 	"github.com/confluentinc/cli/v3/pkg/errors"
+	"net/http"
```

Comment:
We probably want to revert this to maintain our usual style of having line-separated blocks for 1) go standard library, 2) non-Confluent third party dependencies, 3) Confluent dependencies, 4) CLI pkgs.

Reply:
okay, i didnt make this change manually, it happened automatically so i thought it was what was preferred, but will revert it back now.